### PR TITLE
Added Database Schema for product and category

### PR DESCRIPTION
**Pull request by tavisca-cpadole**

### CODE REVIEW

File: DB Schema/category_document.json5 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+category document
+{
+	id: String,
+	name: String,
+	tags:[
+		{type: String},
+		....
+	],
+	createdAt: Date,
+	UpdatedAt: Date
```
Comment (swaghtavisca) [inline DB Schema/category_document.json5]: updatedAt

File: DB Schema/category_document.json5 [hunk 2]
```diff
@@ -0,0 +1,11 @@
+category document
+{
+	id: String,
+	name: String,
+	tags:[
+		{type: String},
```
Comment (swaghtavisca) [inline DB Schema/category_document.json5]: type -> name

File: DB Schema/product_document.json5 [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	expirationDate: Date,
+	price: {
+		currency: String,
+		amount: Double, 
+		isNegotiable: Boolean,
+		},
+	gallery: {
+		heroImageUrl: String,
+		imageUrls:[
+				{type: String},
```
Comment (swaghtavisca) [inline DB Schema/product_document.json5]: type -> url

**Review by swaghtavisca — CHANGES_REQUESTED** [review]
please check

File: DB Schema/product_document.json5 [hunk 4]
```diff
@@ -0,0 +1,32 @@
+product document
+{
+	id: String,
+	userId: String,
```
Comment (achirametavisca) [inline DB Schema/product_document.json5]: Rename it to sellerId

File: DB Schema/product_document.json5 [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+	categoryId: String,
+	name: String,
+	description: String,
+	purchasedDate: Date,
+	createdAt: Date,
```
Comment (achirametavisca) [inline DB Schema/product_document.json5]: Rename it to createdDate

File: DB Schema/product_document.json5 [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+	updatedAt: Date,
```
Comment (achirametavisca) [inline DB Schema/product_document.json5]: Rename it to updatedDate

**Review by achirametavisca — CHANGES_REQUESTED** [review]

File: DB Schema/category_document.json5 (hunk 1 above)
Comment (tavisca-cpadole) [reply DB Schema/category_document.json5]: done

File: DB Schema/category_document.json5 (hunk 2 above)
Comment (tavisca-cpadole) [reply DB Schema/category_document.json5]: done

File: DB Schema/product_document.json5 (hunk 3 above)
Comment (tavisca-cpadole) [reply DB Schema/product_document.json5]: done

File: DB Schema/product_document.json5 (hunk 4 above)
Comment (tavisca-cpadole) [reply DB Schema/product_document.json5]: done

File: DB Schema/product_document.json5 (hunk 5 above)
Comment (tavisca-cpadole) [reply DB Schema/product_document.json5]: done

File: DB Schema/product_document.json5 (hunk 6 above)
Comment (tavisca-cpadole) [reply DB Schema/product_document.json5]: done

**Review by achirametavisca — APPROVED** [review]

**Review by swaghtavisca — APPROVED** [review]
looks OK

**Pull request merged**